client/web/admin/src/views/System/User/Editor.vue

```diff
@@ -140,11 +141,16 @@ export default {
   data () {
     return {
       user: undefined,
+      initialUserState: undefined,
 
       membership: {
         active: [],
         original: [],
```

Comment:
isn't the original already keeping track of the changes? Maybe rename it to initial and use it for that.